pennylane/qchem/openfermion_obs.py

```diff
@@ -945,11 +962,16 @@ def molecular_hamiltonian(
         if args is None:
             h = qml.qchem.diff_hamiltonian(mol, core=core, active=active)()
             coeffs = qml.numpy.real(h.coeffs, requires_grad=False)
-            h = qml.Hamiltonian(coeffs, h.ops, grouping_type=grouping_type, method=grouping_method)
         else:
             h = qml.qchem.diff_hamiltonian(mol, core=core, active=active)(*args)
             coeffs = qml.numpy.real(h.coeffs)
+
+        if active_new_opmath():
```

Comment:
What is the difference between `active_new_opmath` and `enable_new_opmath`?

Reply:
The first returns the current state of the global variable, while the second sets the global variables value to `True`.